docs/running-as-a-service.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Running as a service
+description: "How to run JMusicBot as a service, so that it can run in the background without needing to be manually started."
+---
+
+## Running as a service
+Running JMusicBot as a service allows it to run in the background without needing to be manually started. This is useful for running the bot on a server, or if you want to be able to close your terminal without stopping the bot.
+
+### Linux using systemd
+
+!!! warning
+    This method assumes that you've created a user for the bot to run as. If you haven't, see [this guide](https://www.digitalocean.com/community/tutorials/how-to-create-a-sudo-user-on-ubuntu-quickstart) for instructions.
+
+!!! note
+    Copy the jar file to the home directory of the user that the bot is running as, or change the `WorkingDirectory` and `ExecStart` lines in the service file to point to the correct location.
+
+1. Open a terminal and run the following command to create a new service file:
+
+```bash
+sudo nano /etc/systemd/system/JMusicBot.service
+```
+
+2. Copy the following text into the file and save it:
+
+```ini
+[Unit]
+Description=JMusicBot
+Requires=network.target
+After=network.target
+
+[Service]
+WorkingDirectory=/home/<username>
+User=<username>
+Group=<username>
+Type=simple
+ExecStart=/usr/bin/env java -Dnogui=true -jar JMusicBot.jar
```

Comment:
```suggestion
Environment=nogui=true
ExecStart=/usr/bin/env java -jar JMusicBot.jar
```

Reply:
i f  u wanna be really fancy, u can move the env into the systemd config line instead of executing command